mt.json

```diff
@@ -0,0 +1,10 @@
+{
+  "authors": [
```

Comment:
Though I imagine thorough discussion and planning on all the 'features' of a package manifest should occur at some point. I probably am jumping the gun at this point.